mw/em/code/em_application.cc

```diff
@@ -48,6 +48,7 @@ core::ErrorCode EmApplication::Run(std::stop_token token) {
  */
 core::ErrorCode EmApplication::Initialize(
     const std::unordered_map<std::string, std::string>& parms) {
+  this->exec_.ForceStop();
```

Comment:
czemu stopujesz w inicjalizacji ?

Reply:
bo jeśli em padnie to i tak nic nam to nie da, a ty jak wczytujesz liste aplikacji to nie wczytujesz em jako aplikacji.